jupyter-book/multi_omics/1_0_scATAC_intro.ipynb

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
<blockquote>Another important characteristic of scATAC data is its high sparsity</blockquote><p> </p><p>You added some nice numbers in the paper (10-15% or sth). Think being explicit here is nice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>